.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1207,7 +1210,9 @@ public class ChatCompletionOptions : IJsonModel<ChatCompletionOptions>, IPersist
         public IDictionary<int, int> LogitBiases { get; }
         public int? MaxOutputTokenCount { get; set; }
         public IDictionary<string, string> Metadata { get; }
+        public ChatMessageContent PredictedContent { get; set; }
```

Comment:
Since the "prediction" property has a "type" property, I wonder if we should apply the squishing pattern like we did with `ChatToolCall`. Perhaps something like this:

```csharp
public class ChatPrediction : IJsonModel<ChatPrediction>, IPersistableModel<ChatPrediction> {
    public ChatMessageContent Content { get; }
    public ChatPredictionKind Kind { get; }
    public static ChatPredictionKind CreateContentPrediction(ChatMessageContent content);
}

public enum ChatPredictionKind {
    Content = 0
}
```

The documentation also refers to this type as "static content" (instead of simply "content" like the API does), so replacing "Content" for "StaticContent" everwhere might be a good idea (e.g. "CreateStaticContentPrediction")